codec/src/main/java/io/netty/handler/codec/compression/StandardCompressionOptions.java

```diff
@@ -41,6 +41,20 @@ public static BrotliOptions brotli(Encoder.Parameters parameters) {
         return new BrotliOptions(parameters);
     }
 
+    /**
+     * @see ZstdOptions#DEFAULT
```

Comment:
Add full JavaDoc here. Just swap these JavaDoc.

Reply:
Can you please explain further, what is "swap these JavaDoc"? Thanks

Reply:
I have done the same here: https://github.com/netty/netty/pull/11468/files

Swapped `BrotliOptions` JavaDoc with `StandardCompressionOptions`.

Reply:
I got it, thanks